src/EdgeOrder/readme.md

```diff
@@ -0,0 +1,124 @@
+<!-- region Generated -->
+# Az.EdgeOrder
+This directory contains the PowerShell module for the EdgeOrder service.
+
+---
+## Status
+[![Az.EdgeOrder](https://img.shields.io/powershellgallery/v/Az.EdgeOrder.svg?style=flat-square&label=Az.EdgeOrder "Az.EdgeOrder")](https://www.powershellgallery.com/packages/Az.EdgeOrder/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.EdgeOrder`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/edgeorder/resource-manager/Microsoft.EdgeOrder/stable/2021-12-01/edgeorder.json
```

Comment:
```suggestion
input-file:
# You need to specify your swagger files here.
  - $(repo)/specification/edgeorder/resource-manager/Microsoft.EdgeOrder/stable/2021-12-01/edgeorder.json
branch: input-commit-hash-here
```
Please use the `branch` config to lock the commit hash of the swagger so it's consitent in each generation however swagger changes on main branch.
Example: https://github.com/Azure/azure-powershell/tree/main/src/Aks/Aks.Autorest#autorest-configuration